contrib/babelfishpg_tsql/src/backend_parser/parser.c

```diff
@@ -314,6 +317,15 @@ pgtsql_base_yylex(YYSTYPE *lvalp, YYLTYPE * llocp, core_yyscan_t yyscanner)
 			if (next_token == TSQL_APPLY)
 				cur_token = TSQL_OUTER;
 			break;
+		case ALTER:
+			switch (next_token)
+			{
+				case PROCEDURE:
+				case TSQL_PROC:
```

Comment:
What's the difference between PROCEDURE/TSQL_PROC

Reply:
```
PG_KEYWORD("proc", TSQL_PROC, UNRESERVED_KEYWORD)
PG_KEYWORD("procedure", PROCEDURE, UNRESERVED_KEYWORD)
```

TSQL_PROC is "proc" and PROCEDURE is "procedure"